lit_gpt/config.py

```diff
@@ -595,4 +595,29 @@ def norm_class(self) -> Type:
 configs.extend(freewilly_2)
 
 
+##########################
+# NousResearch Nous-Hermes-Llama2-13b
```

Comment:
There is already `NousResearch` config section at line 482.
Could you put it there? And ideally to have the same order of arguments.

Reply:
Yes sure!